services/federation/follow_calls.go

```diff
@@ -0,0 +1,51 @@
+package federation
+
+import (
+	"github.com/yitsushi/go-misskey/core"
+	"github.com/yitsushi/go-misskey/models"
+)
+
+// FollowRequest is the request used by the followers and following commands.
+type FollowRequest struct {
```

Comment:
Now I see what cause my confusion. It's not FollowRequest, it's Following/Followers request. It's not the same as following someone, FollowRequest sounds like "i want to follow xxxx" which is not that endpoint group.

Maybe just do not export the request, so if someone is browsing the documentation, they will not get confused about it.

Or I have no idea.